spec/javascripts/unit/modules/main-navigation.spec.js

```diff
@@ -5,32 +5,32 @@ describe('Main Menu Block module', function () {
 
   beforeEach(function () {
     var DOM =
-      `<div class="app-b-main-nav" data-module="app-b-main-nav">
```

Comment:
Changes look fine, but the commit structure could be better - individually, each of these commits currently represents a breaking change. In future if the change warrants multiple commits I'd go block by block, but for simplicity now I'd suggest merging all the commits together.